dap/BESDapResponseBuilder.cc

```diff
@@ -120,6 +120,7 @@
 #include "BESStopWatch.h"
 #include "DapFunctionUtils.h"
 #include "RequestServiceTimer.h"
+#include "../dispatch/BESContextManager.h"
```

Comment:
It's better to not use a relative path like this to include the file. You could add `-I$abs_topsrcdir)` to AM_CPPFLAGS in the Makefile.am and then use "dispatch/BESContextManager.h"